### PR TITLE
AI Gen RBA (beta)

### PR DESCRIPTION
**Pull request by jsboak**
Doc for AI Gen RBA (beta) release.  Please look over for clarity and any missing content.

### CODE REVIEW

File: docs/manual/jobs/ai-generated-runbooks.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+To enable _AI-Generated Runbooks_:<br>
+1. Navigate to the AI Manager through the System Menu (gear icon in the upper-right):
+   ![AI Manager Menu](@assets/img/ai-manager-menu.png)
+2. Click the checkbox to `**Enable AI Generated Runbooks**`
+3. Select the projects where the feature should be enabled:
+   ![Enable in Projects](@assets/img/enable-ai-in-project.png)
+4. Click **Save**.
+
+:::warning First Time Configuration
+While the feature is still in beta, users will be asked to review and agree to the beta feature Terms & Conditions. Only **Add Admin** users are allowed to agree to the beta Terms & Conditions.
```
Comment (fdevans) [inline docs/manual/jobs/ai-generated-runbooks.md]: **App Admins**

File: docs/manual/jobs/ai-generated-runbooks.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+While the feature is still in beta, users will be asked to review and agree to the beta feature Terms & Conditions. Only **Add Admin** users are allowed to agree to the beta Terms & Conditions.  
+:::
+
+## Using AI-Generated Runbooks
+
+To use _AI-Generated Runbooks_:<br>
+
+1. Navigate to the **Jobs** tab.
+2. Click on **Generate Job**:
+   ![Generate Job](@assets/img/jobs-page-generate-job.png)
```
Comment (fdevans) [inline docs/manual/jobs/ai-generated-runbooks.md:44]: Did you preview this?  sometimes we need <br/> entries after the images so it's not all squished together.

File: docs/manual/jobs/ai-generated-runbooks.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+   ![Job Preview](@assets/img/ai-gen-rba-job-preview.png)
+5. Review the Job Preview.  If the generated Job is insufficient, either click **Discard** or type in a new prompt.  Otherwise, click on **Continue to Job Editor**.
+6. From the Job editor, modify the Job definition as needed and then click **Create**.
+
+:::warning Error Messages
+Users may encounter the following error message:
+
+1. **`The result data had a problem. You may need to retry your request shortly.`**
+
+In this circumstance, it is ok to try the same prompt again and fairly shortly after the initial prompt query.
```
Comment (fdevans) [inline docs/manual/jobs/ai-generated-runbooks.md]: This sentence reads a bit funny towards the end.

**Review by fdevans — APPROVED** [review]
Approved with these minor changes.

File: docs/manual/jobs/ai-generated-runbooks.md (hunk 2 above)
Comment (jsboak) [reply docs/manual/jobs/ai-generated-runbooks.md:44]: yup - it looked fine in preview to me:

Screen Shot 2023-08-31 at 7 55 32 AM

File: docs/manual/jobs/ai-generated-runbooks.md (hunk 1 above)
Comment (jsboak) [reply docs/manual/jobs/ai-generated-runbooks.md]: fixed in latest commit

File: docs/manual/jobs/ai-generated-runbooks.md (hunk 3 above)
Comment (jsboak) [reply docs/manual/jobs/ai-generated-runbooks.md]: reworded in latest commit

**Pull request merged**